pkg/helper/operator.go

```diff
@@ -0,0 +1,9 @@
+package helper
```

Comment:
Let's not add more to `helper` it's a something that we did wrong at the beginning, let's create a `pkg/environment/` module and put operator.go there, in future we will add the other env vars

It would be use like:

```golang
environment.IsOperator()
```